RPIPs/RPIP-62.md

```diff
@@ -8,27 +8,27 @@ status: Draft
 type: Protocol
 category: Core
 created: 2024-07-25
+requires: rpip-63
 ---
 
 ## Abstract
 This proposal aims to improve RPL tokenomics in the short term before the changes of [RPIP-49](RPIP-49.md) can be implemented.
-Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% and scales linearly with RPL stake up to 14% at 10% of borrowed ETH.
-The cliff for RPL rewards is <explicitly maintained at 10% | removed by extending rewards linearly below 10%> of borrowed ETH. Scrub penalties are changed to be taken out of the node operator's ETH bond instead of slashing RPL.
+Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% for zero RPL staked, and scales linearly up to a max of 14% for RPL staked positions worth at least 10% of borrowed ETH.
```

Comment:
```suggestion
Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% for zero RPL staked, and scales linearly up to a max of 14% for staked RPL positions worth at least 10% of borrowed ETH.
```

Reply:
Definitely an improvement over the current version. Switching it to "staked RPL position" from "RPL staked position" makes it sound a bit less awkward to me.